libs/native-win/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=native-win
```

Comment:
open to suggestion on artifact id @JakeWharton

Reply:
I would probably do whatever aapt2 does for its native artifact

Reply:
I'm not sure what the suggestion based on this: 
`/Users/jrod/.gradle/caches//modules-2/files-2.1/com.android.tools.build/aapt2/4.1.2-6503028/9b3759a6b73cf339d81c99794fb99c402da9a00c/aapt2-4.1.2-6503028-osx.jar`

Can you clarify?

Reply:
`layoutlib-native-win`?

Reply:
I like `layoutlib-native-win`. It's a bit more descriptive.

Reply:
updated!